src/ngraph/frontend/onnx_import/CMakeLists.txt

```diff
@@ -155,6 +155,8 @@ add_library(onnx_import STATIC
         op/relu.hpp
         op/reshape.cpp
         op/reshape.hpp
+		op/reverse_sequence.cpp
```

Comment:
fix indentation

Reply:
Done. I had incorrect settings of IDE ;)